apcd-cms/src/taccsite_cms/urls_custom.py

```diff
@@ -2,5 +2,5 @@
 
 custom_urls = [
     path('administration/', include('apps.admin_regis_table.urls', namespace='administration')),
-    path('register/request-to-submit/', include('apps.submission_form.urls', namespace='apcd'))
+    path('register/', include('apps.submission_form.urls', namespace='apcd'))
```

Comment:
**Semantics.** This suggests that `submission_form.urls` have all the URLs for all the pages under "Register"? **If so**, will all of those forms be "submission" forms? **If not**, why this change?

<details>

There are more pages to come (I don't know if any are forms). H.P. had added the pages, but I overwrite them when I updated QA server with Prod. content. T.B. would know which pages (and forms, if any) are coming under "Register".

</details>

Reply:
That's a good point, and I think it comes down to why I've named the app "submissions form" which is a bit confusing ( because it's actually the registration form ) This can be changed, but I was thinking that there would be multiple endpoints for `/register/`

would it make more sense to keep this structure and change the name of the custom app, or is there even a need for multiple endpoints under `/register/`?